src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Layers/LoadWebTiledLayer/readme.md

```diff
@@ -2,18 +2,24 @@
 
 Display a tiled web layer.
 
-![screenshot](LoadWebTiledLayer.jpg)
+![Image of web tiled layer](LoadWebTiledLayer.jpg)
+
+## Use case
+
+Tiled map services are a set of pre-generated images (e.g. "tiles") arranged in folders for each row, column, and zoom level. As you navigate the map, map tiles are requested for the current extent. `ArcGISTiledLayer` and `WMTSLayer` are types of tiled map services used for specific data types. `WebTiledLayer` is useful for displaying other data sources that contain tiles arranged in a row/column/level directory structure, such as OpenStreetMap.
 
 ## How to use the sample
 
-The web tiled basemap will load automatically when the sample starts.
+Run the sample and a map will appear. As you navigate the map, map tiles will be fetched automatically and displayed on the map.
 
 ## How it works
 
-1. Create a `WebTiledLayer` from a URL and a list of subdomains.
-2. Create a new `Basemap` from the layer.
-3. Update the attribution on the layer. Note: this is a necessary step because web tiled services don't have associated service metadata.
-4. Display the basemap.
+To display the web tiled layer, this sample:
```

Comment:
This might need to be fixed in the base design, but I think the way it was before was closer to the standard format.